newIDE/app/src/EventsSheet/InstructionEditor/SelectorListItems/SelectorInstructionsTreeListItem.js

```diff
@@ -57,6 +57,9 @@ export const renderInstructionOrExpressionTree = <
             primaryText={key}
             selected={selected}
             id={
+              // TODO: This id is used by in app tutorials. When in app tutorials
```

Comment:
I feel that this todo will stay in the code for a while if we don't do it now.
Shall we update the tutorial now?

Reply:
Also, in addition of an id, feel free to use a class name. After all, it's not that different and while an identifier is good, a class name will do the trick too. Just make sure, like identifiers, that is unique and well prefixed so it can't be confused for something else.